src/main/java/lotto/LottoTicket.java

```diff
@@ -1,24 +1,52 @@
 package lotto;
 
-import java.util.List;
+import java.util.*;
+import java.util.stream.Collectors;
 
 public class LottoTicket {
 
-    private List<Integer> lottoNumbers;
+    private static final int DEFAULT_SIZE = 6;
 
-    private LottoTicket(List<Integer> lottoNumbers) {
-        this.lottoNumbers = lottoNumbers;
+    private final Set<LottoNumber> lottoNumbers;
+
+    private LottoTicket(Set<LottoNumber> lottoNumbers) {
+        if (Objects.isNull(lottoNumbers)) {
+            throw new NullPointerException();
+        }
+        validate(lottoNumbers);
+        this.lottoNumbers = new TreeSet<>(lottoNumbers);
     }
 
-    public static LottoTicket of(List<Integer> lottoNumbers) {
+    public static LottoTicket of(Set<LottoNumber> lottoNumbers) {
         return new LottoTicket(lottoNumbers);
     }
 
-    public List<Integer> getLottoNumbers(){
+    public static LottoTicket of(List<Integer> lottoNumbers) {
+        if (Objects.isNull(lottoNumbers)) {
+            throw new NullPointerException();
+        }
+        return of(
+                lottoNumbers.stream()
+                .map(LottoNumber::of)
+                .collect(Collectors.toSet())
+        );
```

Comment:
```suggestion
        return new LottoTicket(
                lottoNumbers.stream()
                .map(LottoNumber::of)
                .collect(Collectors.toSet())
        );
```
생성자를 호출하지 않은 이유가 있을까요 ?